components/components.css

```diff
@@ -21,6 +23,7 @@
     --COLOR-DANGER:#7F1D1D;
     --COLOR-SUCCESS:#14532D;
     --COLOR-GREY:#7e818c;
+    --COLOR-RED:red;
```

Comment:
why to mix RGB and color names, maintain consistency

Reply:
thanks, i changed this